pkg/endpointmanager/conntrack.go

```diff
@@ -29,6 +29,16 @@ import (
 const (
 	// MinGcInterval is the minimum garbage collection interval.
 	MinGcInterval int = 5
+
+	// Global string for refering in metrics
+	Global string = "global"
+
+	// Expires string to refering on metrics
+	expires string = "expires"
+
+	// CleanOnStart string to refering that the CT was clean because it was
+	// cleaning the restored endpoints CT.
+	CleanOnStart string = "cleanOnStart"
```

Comment:
Does this need to be exported?

It would also make this one clearer if the comment states that it's a scope.

Reply:
sorry, didn't push without that :-(